tests/cli-misc.rs

```diff
@@ -568,6 +568,40 @@ fn rename_rls_remove() {
     });
 }
 
+#[test]
+fn install_with_profile() {
+    clitools::setup(Scenario::Full, &|config| {
+        set_current_dist_date(config, "2015-01-01");
+        expect_ok(config, &["rustup", "default", "stable"]);
+
+        // Installing with minimal profile should only install rustc
+        expect_ok(
+            config,
+            &[
+                "rustup",
+                "install",
```

Comment:
I'd have preferred to see `rustup toolchain install` rather than `rustup install` since the latter is a deprecated, hidden, alias.